pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -315,7 +314,7 @@ func (c *APIClient) checkResourcesAuth() error {
 
 // ComponentStatuses returns the component status list from the APIServer
 func (c *APIClient) ComponentStatuses() (*v1.ComponentStatusList, error) {
-	return c.Cl.CoreV1().ComponentStatuses().List(metav1.ListOptions{TimeoutSeconds: &globalTimeoutSeconds})
+	return c.Cl.CoreV1().ComponentStatuses().List(metav1.ListOptions{TimeoutSeconds: &c.timeoutSeconds})
```

Comment:
The `timeoutSeconds` is being used here for more than just the required calls for metadata mapping but the config option is call `kubernetes_metadata_tag_client_timeout`. 

Can we name this something more generic like `kubernetes_apiserver_client_timeout`?

Not a big deal so I'll leave this up to you.

Reply:
You're right it's way better